app/models/manageiq/providers/ibm_cloud/cloud_object_storage/storage_manager.rb

```diff
@@ -0,0 +1,37 @@
+class ManageIQ::Providers::IbmCloud::CloudObjectStorage::StorageManager < ManageIQ::Providers::StorageManager
```

Comment:
Since the vendor here is already `IbmCloud` I wonder if it'd be better (or at least shorter) to call the class name here just: `ManageIQ::Providers::IbmCloud::ObjectStorage::StorageManager`

IBM Cloud Cloud Object Storage is a little redundant